.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,4 +1,10 @@
-[Issue text goes here].
+---
+name: Bug Report
+about: Create a report to help us improve
+
+---
+
+<!-- Issue text goes here. -->
```

Comment:
i'm worried about people putting the issue text inside of the comment on accident, leading to very confusing "empty" issues. Can we go back to [Issue text goes here]?